src/app.js

```diff
@@ -27,6 +28,13 @@ export default class App extends React.Component {
       cardId: this.state.cardId + 1
     })
   }
+  deleteCard() {
+    const id = this.state.flashcards.findIndex(card => card.id)
```

Comment:
Did you push all of your commits? This always deletes the first card with a non-zero id property.